samples/bluetooth/peripheral_hids_mouse/README.rst

```diff
@@ -1,24 +1,129 @@
+
+This sample is similar to Zephyr's :ref:`zephyr:ble_peripheral` sample, except that it specifically exposes the HID GATT Service.
+It uses a report map for a generic mouse.
 
 
 Requirements
 ************
 
-* A board with BLE support
+* One of the following development boards:
+
+  * nRF9160 DK board (PCA10090)
+  * nRF52840 Development Kit board (PCA10056)
+  * nRF52 Development Kit board (PCA10040)
+  * nRF51 Development Kit board (PCA10028)
+
+User interface
+**************
+
+Button 1:
+   Simulate moving the mouse pointer 5 pixels to the left.
+
+Button 2:
+   Simulate moving the mouse pointer 5 pixels upward.
+
+Button 3:
+   Simulate moving the mouse pointer 5 pixels to the right.
+
+Button 4:
+   Simulate moving the mouse pointer 5 pixels downward.
 
-Building and Running
+
+
+Building and running
 ********************
 
 This sample can be found under :file:`samples/bluetooth/peripheral_hids_mouse` in the |NCS| folder structure.
 
 See :ref:`gs_programming` for information about how to build and program the application.
+
+Testing
+=======
+
+After programming the sample to your board, you can test it either by connecting the board as a mouse device to a Microsoft Windows computer or by connecting to it with `nRF Connect for Desktop`_.
+
+Testing with a Microsoft Windows computer
+-----------------------------------------
+
+To test with a Microsoft Windows computer that has a Bluetooth radio, complete the following steps:
+
+1. Power on your development board.
+#. On your Windows computer, search for Bluetooth devices and connect to the device named "NCS HIDS mouse".
+#. Push Button 1 on the board.
+   Observe that the mouse pointer on the computer moves to the left.
+#. Push Button 2.
+   Observe that the mouse pointer on the computer moves upward.
+#. Push Button 3.
+   Observe that the mouse pointer on the computer moves to the right.
+#. Push Button 4.
+   Observe that the mouse pointer on the computer moves downward.
+#. Disconnect the computer from the device by removing the device from the computer's devices list.
+
+
+Testing with nRF Connect for Desktop
+------------------------------------
+
+To test with `nRF Connect for Desktop`_, complete the following steps:
+
+1. Power on your development board.
+#. Connect to the device from nRF Connect (the device is advertising as "NCS HIDS mouse"), then bond to the device.
+   To bond, click the settings button for the device in nRF Connect, select **Pair**, check **Perform Bonding**, and click **Pair**.
+   Wait until the bond is established before you continue.
+   Observe that the services of the connected device are shown.
+#. Click the **Play** button for all HID Report characteristics.
+#. Push Button 1 on the board.
+   Observe that a notification is received on one of the HID Report characteristics, containing the value ``FB0F00``.
+   The format used for mouse reports is the following byte array: ``[Buttons, X-translation, Y-translation]``.
```

Comment:
Can we add an explanation on what the value means? Like "move button 'FB' '0F' pixels to the left and '00' pixels up".